src/ol_social_auth/README.rst

```diff
@@ -0,0 +1,51 @@
+Open Learning Social Auth
+=======================
+
+An Open edX plugin implementing MIT social auth backend
+
+Version Compatibility
+---------------------
+
+Compatible with the Nutmeg release of the Open edX and onwards. May work with prior releases as well.
```

Comment:
No, I copied this from `openedx-companion-auth` Readme. Since both are used together, I thought they would have a similar support. What should I write here? `Compatible with all edx releases`?